tooling/Slack-channels.md

```diff
@@ -0,0 +1,38 @@
+# Slack
```

Comment:
Nit: I'd rename the file `Slack.md` Just given that it covers more than just the channels.